web/src/app/donate/duck/operations.js

```diff
@@ -1,10 +1,15 @@
-import { requestInsertBounty, receiveInsertBountySuccess } from './actions';
+import { requestInsertBounty, receiveInsertBountySuccess, receiveInsertBountyFailure } from './actions';
+import RestClient from '../../RestClient';
 
 const insertBounty = (payload) => (dispatch) => {
   dispatch(requestInsertBounty(payload));
 
-  // TODO: call backend endpoint to insert bounty
-  dispatch(receiveInsertBountySuccess());
+  RestClient.POST('donations', payload, (response) => {
```

Comment:
From what I can tell this isn't being called? Where should this be called in the payment flow?

Reply:
Goodcatch. updated so this is now called during the payment flow